Resort.java

```diff
@@ -183,7 +183,7 @@ public String travel(int pPassId, String ferCode) {
         Pass pass = getPass(pPassId);
         Ferry ferry = getFerry(ferCode);
         if (pass != null && ferry != null) {
-            return ferry.processPass(pass, getIsland(findPassLocation(pPassId)), ferry.getDestinationIsland());
+            return ferry.processPass(pass,ferry.getSourceIsland(), ferry.getDestinationIsland());
```

Comment:
Have space between `pass,` and `ferry.getSourceIsland()`. Also idk why I didn't think of getting the source island like this